.buildkite/scripts/lucene-snapshot/build-snapshot.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -euo pipefail
+
+LUCENE_BRANCH=${LUCENE_BRANCH:-branch_9x}
+
+echo --- Building Lucene snapshot
+
+cd ..
+git clone git@github.com:apache/lucene.git --branch "$LUCENE_BRANCH" --single-branch --depth 1
+cd lucene
+
+LUCENE_SHA=$(git rev-parse --short HEAD)
+echo "Lucene Revision: $LUCENE_SHA"
+
+./gradlew localSettings
+./gradlew clean mavenToLocal -Dversion.suffix="snapshot-$LUCENE_SHA" -Dmaven.repo.local="$(pwd)/build/maven-local"
+
+LUCENE_SNAPSHOT_VERSION=$(ls -d build/maven-local/org/apache/lucene/lucene-core/*/ | xargs basename)
+
+if ! which aws; then
```

Comment:
Should we just add this to our agent images instead?

Reply:
Yep, this is just until that's in place.